[32m2024-02-20 14:03:56 +0100[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 74e5c3a3-9848-4823-bdd5-235267a52062 - 17608 - LOGS_CAPTURED - Started capturing logs in process (pid: 17608).
[32m2024-02-20 14:03:56 +0100[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 74e5c3a3-9848-4823-bdd5-235267a52062 - 17608 - requestToModel - STEP_START - Started execution of step "requestToModel".
[32m2024-02-20 14:03:56 +0100[0m - dagster - [34mINFO[0m - __ASSET_JOB - 74e5c3a3-9848-4823-bdd5-235267a52062 - requestToModel - Data Extraction complete
[32m2024-02-20 14:03:56 +0100[0m - dagster - [34mINFO[0m - __ASSET_JOB - 74e5c3a3-9848-4823-bdd5-235267a52062 - requestToModel -         Datum  Er√∂ffnung  Tageshoch  Tagestief  ...      Umsatz  RSI  EMA  Symbol
0  1999-11-01      68.06      71.88      66.31  ...  12824100.0  0.0  0.0    AMZN
1  1999-11-02      69.75      70.00      65.06  ...  13243200.0  0.0  0.0    AMZN
2  1999-11-03      68.19      68.50      65.00  ...  10772100.0  0.0  0.0    AMZN
3  1999-11-04      67.19      67.19      61.00  ...  16759200.0  0.0  0.0    AMZN
4  1999-11-05      64.75      65.50      62.25  ...  11091400.0  0.0  0.0    AMZN

[5 rows x 9 columns]
[32m2024-02-20 14:03:56 +0100[0m - dagster - [34mERROR[0m - [31m__ASSET_JOB - 74e5c3a3-9848-4823-bdd5-235267a52062 - 17608 - requestToModel - STEP_FAILURE - Execution of step "requestToModel" failed.

dagster._core.errors.DagsterExecutionStepExecutionError: Error occurred while executing op "requestToModel"::

AttributeError: 'Session' object has no attribute 'json'

Stack Trace:
  File "/opt/homebrew/lib/python3.11/site-packages/dagster/_core/execution/plan/utils.py", line 54, in op_execution_error_boundary
    yield
  File "/opt/homebrew/lib/python3.11/site-packages/dagster/_utils/__init__.py", line 467, in iterate_with_context
    next_output = next(iterator)
                  ^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 131, in _coerce_op_compute_fn_to_iterator
    result = invoke_compute_fn(
             ^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 125, in invoke_compute_fn
    return fn(context, **args_to_pass) if context_arg_provided else fn(**args_to_pass)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/marcelthomas/Development/MLOPS/my-dagster-project/quickstart_etl/assets/prod-pipeline.py", line 323, in requestToModel
    context.log.info(f"Response: {session.json()}")
                                  ^^^^^^^^^^^^
[0m
